lib/src/Base/Type/HermitianMatrix.cxx

```diff
@@ -63,6 +63,17 @@ HermitianMatrix::HermitianMatrix(const HermitianMatrix & h)
   // Nothing to do
 }
 
+/* Assignment operator */
+HermitianMatrix & HermitianMatrix::operator=(const HermitianMatrix & rhs)
+{
+  if (this != &rhs)
+  {
+    SquareComplexMatrix::operator=(rhs);
+    hasBeenHermitianized_ = rhs.hasBeenHermitianized_;
+  }
+  return *this;
+}
+
```

Comment:
Why do you have to define the copy constructor explicitly? What is wrong with the one automatically built by the compiler?

Reply:
Its deprecated in newer c++ standard to have only one of copy constructor or copy assignement operators, it's either both or neither